larky/src/test/resources/stdlib_tests/xml/etree/test_elementtree.star

```diff
@@ -609,6 +609,11 @@ def test_namespace_prefix():
     ET._namespace_map.clear()
     ET._namespace_map.update(nsmap)
 
+def test_parse_and_getroot():
```

Comment:
Will it be reasonable to have one test per method?

Reply:
Not really, I actually took it from the original CPython code